### ISSUE TITLE
Dev hasn't been thanked

### ISSUE THREAD

**Issue by williamkray** [BobHasNoSoul/plugin.audio.navidrome#1]
I haven't been able to thank the developer for this great Kodi plugin or send monetary contribution for their work. Discovering this project existed has resolved an issue I've dealt with for months.

Perhaps see if you can advertise the plugin on the navidrome site somehow as I have found very little evidence of this project's existence until now and could have saved considerable time troubleshooting and working around had I known if this sooner.

**Comment by deluan** [comment]
Agree! This is a great effort and I'm adding it to the list of supported clients: https://www.navidrome.org/docs/overview/#apps
